src/main/jbake/content/docs/what.adoc

```diff
@@ -0,0 +1,208 @@
+= Why Hawkular?
+Heiko Rupp
+2015-07-04
+:description: Vision for Hawkular
+:jbake-type: page
+:jbake-status: draft
+
+
+== Why Hawkular?
+
+.NOTE: WHY: what is the purpose, why Hawkular
+
+To be the best tool to monitor JBoss and other Java Middleware helping
+you spending less and more enjoyable time on middleware management.
+
+OR:
+
+Hawkular will reduce the effort and time spent on systems management and monitoring
+and make this task more enjoyable.
```

Comment:
Sorry, I should have made myself clearer :-) I'm not opposed to a OS-level monitoring. I'm just not sure this should be that prominent to be part of the project's main purpose. To me, it should be "to monitor/manage enterprise applications", with the basic OS-level monitoring/managing there for the basic needs on this level.